core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -713,6 +711,11 @@ protected function doReplace($strBuffer, $blnCache)
 					}
 
 					// Do not use StringUtil::specialchars() here (see #4687)
```

Comment:
Did you add this after the comment on purpose? Because the comment actually belongs to line 719.

Reply:
Yes because this if now skips encoding for the title properties.